scripts/import/import_variant_synonyms

```diff
@@ -147,6 +197,356 @@ sub download_PharmGKB{
   return "rsid.tsv";
 }
 
+=head2 download_UniProt
+
+collect current export from UniProt site
+
+=cut 
+
+sub download_UniProt{
+
+  my $http     = HTTP::Tiny->new();
+  my $response = $http->get('https://www.uniprot.org/docs/humsavar.txt');
+  die "Failed to pick up file\n" unless $response->{success};
+
+  open my $out, ">humsavar.txt" ||die "Failed to write data locally : $!\n";
```

Comment:
Not using brackets and the lower precedence of || means that the file open is not checked

Reply:
Thanks for spotting this (again).